apps/codeforafrica/src/lib/api.netlify-cms/seo.js

```diff
@@ -0,0 +1,25 @@
+import { deepmerge } from "@mui/utils";
+import camelcaseKeys from "camelcase-keys";
+
+import getSettings from "./sections/getSettings";
+import { getCollectionBySlug } from "./utils";
+
+export function setSeo(seoUpdate) {
```

Comment:
```suggestion
function createSeo(pageSeo) {
```
1. No need to expose this, it's just an implementation detail.
2. `create`, `initialise`, etc., are better names.